core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -264,6 +270,14 @@ var TopicEditorPage = function() {
     var pageEditorInput = pageEditor.element(by.css('.oppia-rte'));
     await pageEditorInput.click();
     await pageEditorInput.sendKeys(htmlContent);
+    var conceptCardButton = pageEditor.element(by.css('#cke_24'));
+    await conceptCardButton.click();
+    var skillForConceptCard = element.all(
+      by.css('.protractor-test-rte-skill-selector-item')).first();
```

Comment:
Under what circumstances would there be more than one of these elements? In those situations, how do you know the element you want comes first? In general, we try to avoid `.first()` wherever possible because it's usually better to identify the element you want with UI elements like a user would

Reply:
Added a function to add concept card with skill name.